persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_13/modules/clients/multipleassociations_client.bal

```diff
@@ -24,85 +24,75 @@ public client class MultipleAssociationsClient {
             }
         }
-        sql:ExecutionResult result = check self.persistClient.runInsertQuery(value);
+        _ = check self.persistClient.runInsertQuery(value);
         return value;
     }
 
-    remote function readByKey(int key, MultipleAssociationsRelations[] include = []) returns MultipleAssociations|error {
+    remote function readByKey(int key, MultipleAssociationsRelations[] include = []) returns MultipleAssociations|persist:Error {
         return <MultipleAssociations>check self.persistClient.runReadByKeyQuery(MultipleAssociations, key, include);
     }
 
-    remote function read(MultipleAssociationsRelations[] include = []) returns stream<MultipleAssociations, error?> {
-        stream<anydata, error?>|error result = self.persistClient.runReadQuery(MultipleAssociations, (), include);
-        if result is error {
-            return new stream<MultipleAssociations, error?>(new MultipleAssociationsStream((), result));
+    remote function read(MultipleAssociationsRelations[] include = []) returns stream<MultipleAssociations, persist:Error?> {
+        stream<anydata, sql:Error?>|persist:Error result = self.persistClient.runReadQuery(MultipleAssociations, include);
+        if result is persist:Error {
+            return new stream<MultipleAssociations, persist:Error?>(new MultipleAssociationsStream((), result));
         } else {
-            return new stream<MultipleAssociations, error?>(new MultipleAssociationsStream(result));
+            return new stream<MultipleAssociations, persist:Error?>(new MultipleAssociationsStream(result));
         }
     }
 
-    remote function execute(sql:ParameterizedQuery filterClause) returns stream<MultipleAssociations, error?> {
-        stream<anydata, error?>|error result = self.persistClient.runExecuteQuery(filterClause, MultipleAssociations);
-        if result is error {
-            return new stream<MultipleAssociations, error?>(new MultipleAssociationsStream((), result));
+    remote function execute(sql:ParameterizedQuery filterClause) returns stream<MultipleAssociations, persist:Error?> {
+        stream<anydata, sql:Error?>|persist:Error result = self.persistClient.runExecuteQuery(filterClause, MultipleAssociations);
+        if result is persist:Error {
+            return new stream<MultipleAssociations, persist:Error?>(new MultipleAssociationsStream((), result));
         } else {
-            return new stream<MultipleAssociations, error?>(new MultipleAssociationsStream(result));
+            return new stream<MultipleAssociations, persist:Error?>(new MultipleAssociationsStream(result));
         }
     }
 
-    remote function update(MultipleAssociations value) returns error? {
-        map<anydata> filter = {"id": value.id};
-        _ = check self.persistClient.runUpdateQuery(value, filter);
+    remote function update(record {} value) returns persist:Error? {
+        _ = check self.persistClient.runUpdateQuery(value);
         if value["profile"] is record {} {
```

Comment:
```suggestion
        if value.profile is record {} {
```

Reply:
check the update API. create API looks good

Reply:
We cannot update these two instances due to the following error
```
ERROR [multipleassociations_client.bal:(78:12,78:25)] invalid field access: 'profile' is not a required field in record 'record {| anydata...; |}', use member access to access a field that may have been specified as a rest field
ERROR [multipleassociations_client.bal:(79:46,79:59)] invalid field access: 'profile' is not a required field in record 'record {| anydata...; |}', use member access to access a field that may have been specified as a rest field
ERROR [multipleassociations_client.bal:(83:12,83:22)] invalid field access: 'user' is not a required field in record 'record {| anydata...; |}', use member access to access a field that may have been specified as a rest field
ERROR [multipleassociations_client.bal:(84:37,84:47)] invalid field access: 'user' is not a required field in record 'record {| anydata...; |}', use member access to access a field that may have been specified as a rest field
ERROR [profile_client.bal:(66:12,66:22)] invalid field access: 'user' is not a required field in record 'record {| anydata...; |}', use member access to access a field that may have been specified as a rest field
ERROR [profile_client.bal:(67:37,67:47)] invalid field access: 'user' is not a required field in record 'record {| anydata...; |}', use member access to access a field that may have been specified as a rest field
```

Reply:
Updated in [636eb9d](https://github.com/ballerina-platform/persist-tools/pull/33/commits/636eb9d7e83da8442d99429fa3b92533ed622df9)